spec/features/users/user_can_see_its_stats_page_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
```

Comment:
I would change the path/filename of this test to the following:
`spec/features/users/profile/stats/index_spec.rb`